modules/ROOT/pages/dataweave-formats-multipart.adoc

```diff
@@ -576,10 +579,12 @@ This format accepts properties that provide instructions for reading input data.
 
 [cols="1,1,1,3a", options="header"]
 |===
-| Parameter | Type | Default | Description
-| `boundary` | `String` | `null` | The multipart `boundary` value. A string to
-delimit parts.
-|`defaultContentType`  |`String`| This property has no default value. | Sets the default Content-Type to use on parts of the `multipart/*` format. When set, this property takes precedence over the https://docs.mulesoft.com/dataweave/latest/dataweave-system-properties[system property setting] for `com.mulesoft.dw.multipart.defaultContentType`. _Introduced in DataWeave 2.3 (2.3.0-20210720) for the August 2021 release of Mule 4.3.0-20210719._
+|Parameter |Type |Default|Description
+|`boundary`  |`String`|`null`|The multipart `boundary` value, a string to delimit parts.
+|`defaultContentType`  |`String`|`'application/octet-stream'`|Sets the default Content-Type to use on parts of the `multipart/*` format. When set, this property takes precedence over the setting for the https://docs.mulesoft.com/dataweave/latest/dataweave-system-properties[system property] `com.mulesoft.dw.multipart.defaultContentType`.
+
+Introduced in DataWeave 2.3 (2.3.0-20210720) for the August 2021 release of Mule 4.3.0-20210719.
```

Comment:
TODO: can we add the underline formatting to the source?

Reply:
Updating data-weave master branch via https://github.com/mulesoft/data-weave/pull/2042/files
Also need to update support/2.5.0

Reply:
Added fixed formatting in changed link to xref in https://github.com/mulesoft/data-weave/pull/2042/files

Reply:
fixed via 2042